microsoft-edge/webview2/how-to/winrt-from-js.md

```diff
@@ -657,9 +677,9 @@ Next, pass the WinRT object from the native side of the host app to the web side
 
 ##### [WinUI 2 (UWP)](#tab/winui2)
 
-1. In Solution Explorer, expand the main project, such as **webview2_sample_uwp**, and then select **MainPage.xaml.cs**.
+1. In Solution Explorer, expand the main project, such as **MyUWPGetStartApp**, expand **MainPage.xaml**, and then select **MainPage.xaml.cs**.
 
-1. Below the `MainPage` constructor, add the following `InitializeWebView2Async` method:
+1. Below the `MainPage(){...}` constructor, add the following `InitializeWebView2Async` method:
```

Comment:
Is this change required?

Reply:
The change is not technically required, and is intended to add clarity without adding confusion.

Here is the code listing in PR 172, that shows the braces in question, the brace which terminates the ctor:

https://github.com/mikehoffms/WebView2Samples/blob/user/mikehoffms/getstart-winui2-initial/GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/MainPage.xaml.cs

Reply:
@peiche-jessica - keep or remove?:   (){...}

Reply:
It should be fine without the `(){...}` as there is only one constructor in the code.

Reply:
removed change